app/gui/src/config.rs

```diff
@@ -110,26 +112,27 @@ impl BackendService {
 #[derive(Clone, Debug, Default)]
 pub struct Startup {
     /// The configuration of connection to the backend service.
-    pub backend:       BackendService,
+    pub backend:         BackendService,
     /// The project name we want to open on startup.
-    pub project_name:  Option<ProjectName>,
+    pub project_to_open: Option<ProjectToOpen>,
```

Comment:
Not sure if you realized (the docs here weren't helpful TBH), but the option not always is "project to open" - in cloud (i.e. when we receive LS endpoints instead of PM endpoint) this is a name of project we connect to - so not _project to open_ (as it is, technically, already opened), but _project name_.